tests/common/type_coverage.h

```diff
@@ -14,14 +14,14 @@
 
 #include <sycl/sycl.hpp>
 
+#include "../../util/type_traits.h"
+
 /**
  * @brief Retrieve type name; by default just forward the given one
  */
 template <typename T>
 struct type_name_string {
-    static std::string get(std::string dataType) {
-        return dataType;
-    }
+  static std::string get(std::string dataType) { return dataType; }
```

Comment:
@andreyromanof, please, refrain from adding code formatting changes for the parts, which logic you don't modify. It makes code review difficult. Please, move such changes to a separate PR. It would much easier to review this patch if there is a clear separation between code formatting (non-functional) changes and changes "improving" `type_coverage.h`.

Reply:
I moved format changes to separate PR (#308), to simplify the review process for this PR.

Reply:
Do you mean we should hold reviews until #308 is merged?

Reply:
I think we should because of the large changes in format. 
I converted PR to draft until #308 is merged.